docs/troubleshoot-ksql.rst

```diff
@@ -1,64 +1,44 @@
-.. _troubleshoot-ksql:
+check.. _troubleshoot-ksql:

```

Comment:
this is improper syntax for RST. should be changed back to original.

Reply:
Fixed minor change and moved to the the 5.1.0-post branch.

Reply:
Hey, I changed to the PR to the 5.1.0-post branch, but something doesn't look right to me. If you need to close this PR, I start over and push to 5.1.0-post instead of master.